compose/config/fields_schema_v2.json

```diff
@@ -41,6 +41,13 @@
             "^.+$": {"type": ["string", "number"]}
           },
           "additionalProperties": false
+        },
+        "external": {
+          "type": ["boolean", "object"],
+          "properties": {
+            "name": {"type": "string"}
+          },
+          "additionalProperties": false
```

Comment:
Oh, I just realized, I think we're missing a constraint. We want external and driver/driver_opts to be mutually exclusive, right ?

I think you can set a constraint that if external is used, driver and driver_opts are invalid.